src/components/checkbox/Checkbox.tsx

```diff
@@ -101,45 +127,49 @@ const CheckboxRoot = styled(RadixCheckbox.Root)`
   justify-content: center;
   flex-shrink: 0;
 
-  &[aria-disabled='true'] {
-    color: var(--echoes-color-text-disabled);
-    background-color: var(--echoes-color-background-accent-weak-disabled);
-    border: var(--echoes-color-border-disabled);
+  &[aria-disabled] {
+    color: var(--echoes-color-icon-disabled);
+    background-color: var(--echoes-color-background-disabled);
+    border-color: var(--echoes-color-border-disabled);
 
     &[data-state='checked'],
     &[data-state='indeterminate'] {
-      background-color: var(--echoes-color-background-accent-default-disabled);
+      background-color: var(--echoes-color-background-default-disabled);
+      border-color: var(--echoes-color-background-default-disabled);
     }
 
     &:focus,
     &:focus-visible {
-      outline: var(--echoes-color-border-disabled) solid 2px;
-      outline-offset: var(--echoes-dimension-space-25);
+      outline: var(--echoes-color-border-disabled) solid var(--echoes-focus-border-width-default);
```

Comment:
didn't @marciopmoreira6 say that the focus should always look the same?

Reply:
Indeed, I'll fix it!